### ISSUE TITLE
`cap add android` is missing chalk module

### ISSUE THREAD

**Issue by ndarilek** [ionic-team/capacitor#478]
Using the latest Capacitor packages, I get the following when running `cap add android`:

```
✖ Installing android dependencies: Cannot find module 'chalk'                                                                                                                                                      
✖ add: Cannot find module 'chalk'                                                                                                                                                                                  
```

Possibly of relevance is that I'm using Yarn, not NPM. I'm also using:

`yarn cap add android`

This is a brand new app, created 10 minutes ago, and with the newest Capacitor packages as of then.

Thanks!

**Comment by ndarilek** [comment]
I'm not familiar with npx, but:

`yarn somecommand`

should be equivalent to either running `somecommand` from "scripts", or
node_modules/.bin/somecommand if that isn't found.

**Comment by ndarilek** [comment]
FWIW, `./node_modules/.bin/cap add android` results in the same error.

**Comment by janpio** [comment]
How did you install the Capacitor CLI?

**Comment by ndarilek** [comment]
`|yarn add @capacitor/core @capacitor/cli`

Equivalent of `npm install --save`.
|

**Comment by mlynch** [comment]
if you `yarn add chalk` does it work?

**Comment by ndarilek** [comment]
No, but if I `rm -rf node_modules && npm install` then it works. The
variable seems to be my use of Yarn over NPM. In a project where you
never use NPM, `cap add` doesn't seem to work.

Would be nice if Yarn was supported. It's significantly faster, has a
few developer-friendly touches that make it easier to work with, and I
haven't seen the kinds of bugs in it that I have with NPM recently. I'll
use NPM for the time being.

**Comment by IT-MikeS** [comment]
I have more issues with yarn then with npm, I find windows and yarn really don't get along and on mac I still have issues with yarn, honestly as of npm 5 I no longer use yarn, and I've never looked back lol
